corehq/apps/registration/utils.py

```diff
@@ -108,9 +107,6 @@ def request_new_domain(request, form, is_new_user=True):
         # Avoid projects created by dimagi.com staff members as self started
         new_domain.internal.self_started = not current_user.is_dimagi
 
-        if form.cleaned_data.get('domain_timezone'):
-            new_domain.default_timezone = form.cleaned_data['domain_timezone']
```

Comment:
Why was this removed?

Reply:
I mention the reason both in the commit message and the PR description.
>  I also noticed that domain_timezone was not a field in any of the forms for the two usages (nor is it a field in ANY form), so I removed it.

Reply:
Sorry, missed that.